siriuspy/siriuspy/pwrsupply/bsmp/constants.py

```diff
@@ -12,7 +12,7 @@
 # version of the BSMP implementation of power supplies that is compatible
 # with the current implemenation of this module.
 
-__version__ = 'FRWM - V1.31 V0.41u - 8/19/2020'
+__version__ = 'FRWM - V1.33 V0.41u - 31/09/2020'
```

Comment:
is 31/09/2020 right?

Reply:
@gabrielbrunheira  changed the formatting in the spec (excel file). I justed copied it so as to mirror the primary source of documentation.

Reply:
There was indeed a mistake...it should have been 8/31/2020..I will update this file and send by email.

Reply:
I suggest use the newest version, **v1.36 V0.41 04/09/2020**